src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1563,6 +1577,118 @@ def _get_linux_multicontainer_encoded_config_from_file(file_name):
     # Decode base64 encoded byte array into string
     return b64encode(config_file_bytes).decode('utf-8')
 
+def update_deployment_configs(cmd, resource_group_name, name,
+                              deployment_storage_name=None,
+                              deployment_storage_container_name=None, deployment_storage_auth_type=None,
+                              deployment_storage_auth_value=None, slot=None):
+    
+    # logger.warning("deployment_storage_name: %s deployment_storage_container_name: %s", deployment_storage_name, deployment_storage_container_name)
+    logger.warning("deployment_storage_auth_type: %s deployment_storage_auth_value: %s", deployment_storage_auth_type, deployment_storage_auth_value)
+    
+    if (deployment_storage_name == None or deployment_storage_container_name == None):
```

Comment:
I do not think this is a valid check because customers might just want to update their deployment auth info, not the deployment storage info itself. I would change this to:
```suggestion
    if (deployment_storage_name is not None) != (deployment_storage_container_name  is not None):
```

Reply:
done